src/pheval/analyse/generate_summary_outputs.py

```diff
@@ -142,29 +142,31 @@ def merge_results(result1: dict, result2: dict) -> dict:
 def generate_gene_rank_comparisons(comparison_ranks: [tuple]) -> None:
     """Generate the gene rank comparison of two result directories."""
     for pair in comparison_ranks:
+        result1, result2 = pair
```

Comment:
Is this pythonic? I understand the idea, but wondering if it's generally recommended (point of failure if pair changes structure).

Reply:
The most pythonic way is by accessing the index of the element, so I should probably go back to that